src/Microsoft.IIS.Administration.Core/Versioning/ResourceInfoAttribute.cs

```diff
@@ -46,7 +49,7 @@ public static string ContentType(string name, string version, string previous =
             }
             else {
                 type.Append(".");
-                type.Append(previous);
+                type.Append(previous.StartsWith(applicationTypePrefix) ? previous.Substring(applicationTypePrefix.Length) : previous);
```

Comment:
Is there any chance the previous could be "**A**pplication"?

Reply:
It shouldn't, but I'm happy to change it to case insensitive for future-proofing.

Reply:
If two different cases are semantically the same, I would.